pytorch_lightning/core/step_result.py

```diff
@@ -128,6 +128,7 @@ def log(
         sync_dist_group: Optional[Any] = None,
         sync_fn: Callable = None,
         dataloader_idx: Optional[int] = None,
+        device: torch.device = None,
```

Comment:
let's add an explanation to the docs

Reply:
Not sure I understand this comment. Result.log is an internal function used by LightningModule.log. It shouldn't be used by a user.